src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.ManagedIdentity.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Abstractions;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.AppConfig;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Portion of the TokenAcquisition class that handles logic unique to managed identity.
+    /// </summary>
+#if NETSTANDARD2_0 || NET462 || NET472
+        internal partial class TokenAcquisition : ITokenAcquisitionInternal
```

Comment:
I don't think that you need to re-write here that it implements this interface. Another partial already said it. I believe you can get rid of lines 17-19 included